packages/charInfo_width/charInfo_width.0.1.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "zandoye@gmail.com"
+authors: [ "ZAN DoYe" ]
+homepage: "https://bitbucket.org/zandoye/charinfo_width/"
+bug-reports: "https://bitbucket.org/zandoye/charinfo_width/issues"
+license: "MIT"
+dev-repo: "hg://https://bitbucket.org/zandoye/charinfo_width"
+build: [
+  [make]
```

Comment:
The makefile simply invokes dune build. I would recommend here to replace `[make]` with the standard invocation:
```
[“dune” “build” “-p” name “-j” jobs]
["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Reply:
Note that to enable the tests you need to add also `ppx_expect` as a test dependency